test/suites/device-virtual/main_test.go

```diff
@@ -51,4 +51,9 @@ func TestCommon(t *testing.T) {
 	utils.TestPackaging(t, deviceVirtualSnap, utils.Packaging{
 		TestSemanticSnapVersion: true,
 	})
+
+	utils.TestAutoStart(t, deviceVirtualSnap, utils.Autostart{
+		TestAutoStart: true,
+	})
+
```

Comment:
I think the autostart can be part of TestConfig because it is related to the configuration. Not to be confused with the `config` options.